ToDo.Core.Test/AddViewModelTest.cs

```diff
@@ -85,5 +90,53 @@ public async Task WhenAddingATodo_WithoutTitle_ShowsAlertAndDoesNotAddTheTodo()
 
             Assert.AreEqual(State.Pending, todo.State);
         }
+
+        [TestMethod]
+        public async Task WhenTakingAPicture_PhotoAppearsOnScreen()
+        {
+            // ARRANGE
+            var testPhoto = new MediaFile("", () => new MemoryStream(new byte[] {}));
+            _mediaService.Setup(s => s.CapturePhoto()).ReturnsAsync(testPhoto);
+
+            _computerVisionService.Setup(s => s.GetImageSummary(testPhoto)).ReturnsAsync("Recognized Summary");
+            _computerVisionService.Setup(s => s.RecognizeText(testPhoto)).ReturnsAsync("Recognized Text");
+            _computerVisionService.Setup(s => s.RecognizeTags(testPhoto)).ReturnsAsync(new List<ImageTag>());
+            _computerVisionService.Setup(s => s.RecognizeObjects(testPhoto)).ReturnsAsync(new List<DetectedObject>());
+            _computerVisionService.Setup(s => s.RecognizeBrands(testPhoto)).ReturnsAsync(new List<DetectedBrand>());
+            _computerVisionService.Setup(s => s.RecognizeFaces(testPhoto)).ReturnsAsync(new List<FaceDescription>());
+
+            // ACT
+            await _viewModel.TakePhoto();
+
+            // ASSERT
+            _mediaService.Verify(s => s.CapturePhoto(), Times.Once());
+            Assert.IsNotNull(_viewModel.ImageSource);
+        }
+
+        [TestMethod]
+        public async Task WhenTakingAPicture_AnalyzesPhoto()
+        {
+            // ARRANGE
+            var testPhoto = new MediaFile("", () => new MemoryStream(new byte[] { }));
+            _mediaService.Setup(s => s.CapturePhoto()).ReturnsAsync(testPhoto);
+
+            _computerVisionService.Setup(s => s.GetImageSummary(testPhoto)).ReturnsAsync("Recognized Summary");
+            _computerVisionService.Setup(s => s.RecognizeText(testPhoto)).ReturnsAsync("Recognized Text");
+            _computerVisionService.Setup(s => s.RecognizeTags(testPhoto)).ReturnsAsync(new List<ImageTag>());
+            _computerVisionService.Setup(s => s.RecognizeObjects(testPhoto)).ReturnsAsync(new List<DetectedObject>());
+            _computerVisionService.Setup(s => s.RecognizeBrands(testPhoto)).ReturnsAsync(new List<DetectedBrand>());
+            _computerVisionService.Setup(s => s.RecognizeFaces(testPhoto)).ReturnsAsync(new List<FaceDescription>());
+
+            // ACT
+            await _viewModel.TakePhoto();
+
+            // ASSERT
+            _computerVisionService.Verify(s => s.GetImageSummary(testPhoto), Times.Once());
+            _computerVisionService.Verify(s => s.RecognizeText(testPhoto), Times.Once());
+            _computerVisionService.Verify(s => s.RecognizeTags(testPhoto), Times.Once());
+            _computerVisionService.Verify(s => s.RecognizeObjects(testPhoto), Times.Once());
+            _computerVisionService.Verify(s => s.RecognizeBrands(testPhoto), Times.Once());
+            _computerVisionService.Verify(s => s.RecognizeFaces(testPhoto), Times.Once());
+        }
```

Comment:
As the two tests have the same "Arrange" and the same "Act", they could be merged (just combine the Asserts of both).
Having both of them does not make sense